runtime/compiler/trj9/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -12477,7 +12477,7 @@ J9::Z::CodeGenerator::inlineDirectCall(
       }
 #endif
 
-   if (!comp->getOption(TR_DisableSIMDStringCaseConv) && cg->getSupportsVectorRegisters())
+   if (cg->getSupportsSIMDStringCaseConv() && !comp->getOption(TR_DisableSIMDStringCaseConv))
```

Comment:
Can we get rid of the `!comp->getOption(TR_DisableSIMDStringCaseConv)` check? This check should be done in `CodeGenerator` constructor where we call `setSupportsSIMDStringCaseConv`.